ironfish/src/node.ts

```diff
@@ -93,13 +91,12 @@ export class IronfishNode {
     this.chain = chain
     this.strategy = strategy
     this.metrics = metrics
-    this.miningManager = new MiningManager({ chain, memPool, node: this, telemetry })
+    this.miningManager = new MiningManager({ chain, memPool, node: this })
     this.memPool = memPool
     this.workerPool = workerPool
     this.rpc = new RpcServer(this)
     this.logger = logger
     this.pkg = pkg
-    this.telemetry = telemetry
```

Comment:
Is there a reason telemetry is being removed here? It's a core system, and all core systems are accessible from the node to our node and SDK consumers.